cmd/schema/scan.go

```diff
@@ -0,0 +1,80 @@
+package schema
+
+import (
+	"fmt"
+
+	"github.com/bmeg/grip/gripql"
+	"github.com/bmeg/grip/protoutil"
+	"github.com/bmeg/grip/util"
+)
+
+type edgeDstKey struct {
+	label, to string
+}
+
+type edgeDstMap map[edgeDstKey]interface{}
+
+// ScanSchema attempts to construct a schema of a graph by sampling vertices and edges
+func ScanSchema(conn gripql.Client, graph string, sampleCount uint32) (*gripql.Graph, error) {
+
+	labelQuery := gripql.V().Distinct([]string{"_label"}).Render([]string{"_label"})
```

Comment:
This call is currently super slow since it essentially streams every vertex out of the database... we should consider adding a list labels endpoint.